pyuvsim/simsetup.py

```diff
@@ -199,26 +208,28 @@ def read_votable_catalog(gleam_votable, input_uv=None, source_select_kwds={}, re
     """
     Creates a list of pyuvsim source objects from a votable catalog.
 
+    Tested on: GLEAM EGC catalog, version 2
+
     Args:
         gleam_votable: Path to votable catalog file.
         input_uv: The UVData object for the simulation (needed for horizon cuts)
-        return_table: If True, return the astropy table instead of a list of Source objects. (Default False)
+        return_table: bool, optional
+            Whether to return the astropy table instead of a list of Source objects.
         source_select_kwds: Dictionary of keywords for source selection
             Valid options:
             |  lst_array: For coarse RA horizon cuts, lsts used in the simulation [radians]
-            |  latitude_deg: Latitude of telescope in degrees. Used for declination coarse horizon cut.
-            |  horizon_buffer: Angle (float, in radians) of buffer for coarse horizon cut. Default is about 10 minutes of sky rotation.
-            |                  (See caveats in simsetup.array_to_skymodel docstring)
+            |  latitude_deg: Latitude of telescope in degrees. Used for declination coarse
+            horizon cut.
+            |  horizon_buffer: Angle (float, in radians) of buffer for coarse horizon cut.
+            |      Default is about 10 minutes of sky rotation. (See caveats in
+            simsetup.array_to_skymodel docstring)
```

Comment:
This docstring isn't rendering properly on RTD: https://pyuvsim.readthedocs.io/en/code-format/classes.html#pyuvsim.simsetup.read_votable_catalog

Reply:
Thanks, converted that to a list